k8s/voucher/manifest/application.yaml.template

```diff
@@ -0,0 +1,22 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "$name"
+  namespace: "$namespace"
+  annotations:
+    marketplace.cloud.google.com/deploy-info: '{"partner_id": "partner", "product_id": "Voucher", "partner_name": "Google"}'
```

Comment:
@armandomiani Do we need to do anything with `partner_id` here?

Reply:
partner_id = click-to-deploy-containers
product_id = voucher
partner_name = Google Click to Deploy

Reply:
ack